healpy/pixelfunc.py

```diff
@@ -982,12 +970,7 @@ def nside2resol(nside, arcmin=False):
     >>> hp.nside2resol(256)
     0.0039973699529159707
 
-    >>> hp.nside2resol(7)
-    Traceback (most recent call last):
```

Comment:
Also, let's keep this check but assert that it gives the correct answer instead of raising an exception.